docs/src/main/sphinx/release/release-359.md

```diff
@@ -0,0 +1,71 @@
+# Release 359 (XX Jul 2021)
+
+## General
+
+* Add support for row pattern recognition in window specification. ({issue}`8141`)
+* Add support for {doc}`/sql/set-time-zone`. ({issue}`8112`)
+* Add {func}`geometry_nearest_points`. ({issue}`8280`)
+* Add support for `varchar`, `varbinary` and `date` types to {func}`make_set_digest`. ({issue}`8295`)
+* Add support for granting `UPDATE` privileges. ({issue}`8279`)
+* List materialized view columns in `information_schema.columns` table ({issue}`8113`)
+* Return view's and materialized view's comments in `system.metadata.table_comments`. Previously, the 
+  views and materialized views were reported as if having no comment. ({issue}`8327`)
+* Reduce query memory footprint. ({issue}`8086`)
+* Fix query failure for certain queries with `ORDER BY ... LIMIT` on sorted data. ({issue}`8184`)
+* Fix incorrect query results for certain queries with `WHERE` using `LIKE` with pattern against 
+  `char` columns. ({issue}`8311`)
+* Fix planning failure when using {func}`hash_counts`. ({issue}`8248`)
+* Fix error message when grouping expressions in ``GROUP BY`` queries contain aggregations, window functions or grouping operations. ({issue}`8247`)
+
+## Security
+
+* Fix user mapping for password authentication. Formerly, user mapping for password authentication 
+  would result in a spurious impersonation check. ({issue}`7027`)
+* Fix handling of multiple LDAP user bind patterns. ({issue}`8134`)
+
+## Web UI
+
+* Show session timezone in query details page. ({issue}`4196`)
+
+## BigQuery connector
+
+* Add `bigquery.views-cache-ttl` config property to allow configuring the cache expiration for BigQuery views. ({issue}`8236`)
+* Fix incorrect results when accessing BigQuery records with wrong index. ({issue}`8183`)
+
+## Elasticsearch connector
+
+* Fix potential incorrect results when queries contain an `IS NULL` predicate. ({issue}`3605`)
+* Fix failure when multiple indexes share the same alias. ({issue}`8158`)
+
+## Hive connector
+
+* Add support for Hive views which use `GROUP BY` over a subquery that also uses `GROUP BY` on matching columns. ({issue}`7635`)
+* Add support for granting `UPDATE` privileges when `hive.security=sql-standard` is used. ({issue}`8279`)
+* Fix query failure when reading from a non-ORC insert-only transactional table. ({issue}`8259`)
+* Correct how delete markers are interpreted for ORC ACID tables,
+  so Trino is not filtering out extraneous rows for certain tables. ({issue}`8208`)
+* Improve performance when reading partition information from Glue metastore. ({issue}`8320`)
+* Respect `hive.metastore.glue.get-partition-threads` configuration property. ({issue}`8320`)  
```

Comment:
These two are actually the same issue. Maybe merge the bullets? Since this was technically a regression I'm in favour of keeping 2nd bullet and dropping the first.